### ISSUE TITLE
Creates tunnel on remote side (as root), but does not bring interface up or set ip

### ISSUE THREAD

**Issue by cschech** [danfruehauf/NetworkManager-ssh#99]
Using the Debian package, once the VPN is set up the client side has tun0 configured correctly with the default IP and all the routes are correct but remotely I have to manually run:

ip link set dev tun100 up
ip addr add 172.16.40.1/30 dev tun100

So tun100 is created by NetworkManager-ssh on the remote side but not configured. Unsure why this fails.

I am logging in as root on the remote side so there is no problem there. 

Has anyone else run into this?

**Comment by boris-so** [comment]
> Has anyone else run into this?

Yep, verify you have `ifconfig` util installed on both server and client. I bielive this plugin does not use the modern `ip` tool, but the old one instead.

**Comment by boris-so** [comment]
> installed on both server and client

Not both but server only.
